deps.edn

```diff
@@ -1,4 +1,4 @@
 {:paths ["src/cljs"]
- :deps  {org.clojure/clojurescript {:mvn/version "1.10.520"}
-         reagent                   {:mvn/version "0.8.1"}
+ :deps  {org.clojure/clojurescript {:mvn/version "1.10.773"}
+         reagent                   {:mvn/version "1.1.0"}
          cljsjs/codemirror         {:mvn/version "5.44.0-1"}}}
```

Comment:
Is there a newer cljsjs package for cm?

Reply:
@milt there is not :( It's the main thing i wanted to do but there is definitely not.

Reply:
womp womp